oemof/solph/components.py

```diff
@@ -247,7 +246,7 @@ class GenericStorageBlock(SimpleBlock):
 
     Storage balance :attr:`om.Storage.balance[n, t]`
         .. math:: E(t) = &E(t-1) \cdot
-            (1 - \delta(t))) \\
+            (1 - \delta(t))^{\tau(t)/(1 h)} \\
```

Comment:
1 hour is the default time unit, but not the only possible one. Maybe, introduce a symbol t_u for 1 time unit.

Reply:
At the moment the unit for the time increment is hour by default. But according to the discussion above we should make the API cleaner. So I agree that per time unit is better than per hour. It is possible to use seconds for the time increment to get Joul a energy unit instead of Wh.